src/blog/kuadrant-v1.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Kuadrant v1
+date: 2024-11-28
+author: Jason Madigan
+---
+
+**Update:** Kuadrant [v1.0.1](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.1) is now available, with some small bugfixes.
```

Comment:
`v1.0.2` is in the oven :shrug:

Reply:
yeah, I'm happy to merge this either now, or do a small update PR when we're baked